netlify.toml

```diff
@@ -1,6 +1,6 @@
 [build]
   base    = "docs/"
   publish = "docs/public/"
-  command = "./generate-docs.sh"
+  command = "chmod +x ./generate-docs.sh"
```

Comment:
This won't work as it just changes the mode, it doesn't execute the script.
I believe you can change the mode in git itself. Change the permission on your machine and commit the file.